packages/cardano-services/src/Metadata/util.ts

```diff
@@ -0,0 +1,16 @@
+import { Cardano } from '@cardano-sdk/core';
+import { TxMetadataByHashes } from './DbSyncMetadataService';
+import { TxMetadataModel } from './types';
+import { mapTxMetadata } from './mappers';
+
+export const mapTxMetadataByHashes = (listOfMetadata: TxMetadataModel[]): TxMetadataByHashes => {
+  const metadataMap: Map<Cardano.TransactionId, TxMetadataModel[]> = new Map();
+
+  for (const metadata of listOfMetadata) {
+    const txId = metadata.tx_id.toString('hex') as unknown as Cardano.TransactionId;
```

Comment:
This should be enough since 5ff77c8a8a4cce5c521c39afd9f23d546a5bfa1f
```suggestion
    const txId = metadata.tx_id.toString('hex') as Cardano.TransactionId;
```

Reply:
4cb75f987 should solve this